src/addons/dragAndDrop/WeekWrapper.js

```diff
@@ -268,27 +258,34 @@ class WeekWrapper extends React.Component {
   }
 
   render() {
-    const { children, accessors } = this.props
-
-    let { segment } = this.state
-
     return (
-      <div className="rbc-addons-dnd-row-body">
-        {children}
-
-        {segment && (
-          <EventRow
-            {...this.props}
-            selected={null}
-            className="rbc-addons-dnd-drag-row"
-            segments={[segment]}
-            accessors={{
-              ...accessors,
-              ...dragAccessors,
-            }}
-          />
-        )}
-      </div>
+      <DragAndDropContext.Consumer>
```

Comment:
this should use `static contextType` where possible to avoid the extra component and instance assigning in the render fn